packages/react-components/src/theming/icons.tsx

```diff
@@ -235,6 +237,10 @@ export const DEFAULT_COMPONENT_ICONS = {
   ControlButtonParticipantsContextualMenuItem: <People20Regular />,
   ControlButtonScreenShareStart: <ShareScreenStart20Filled />,
   ControlButtonScreenShareStop: <ShareScreenStop20Filled />,
+  /* @conditional-compile-remove(raise-hand) */
+  ControlButtonRaiseHand: <HandLeft16Regular />,
```

Comment:
We usually use the size 20 icons for the control bar is there a reason we went with the 16 here?